src/transformState.js

```diff
@@ -5,7 +5,36 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+
+  for (const curentAction of actions) {
+    switch (curentAction.type) {
```

Comment:
optional impovements:
1. `action` instead of `curentAction` will be enought (especially in this word a typo)
2. use destructuring assignment